Makefile

```diff
@@ -19,8 +19,7 @@ endef
 
 CPPFLAGS := $(CPPFLAGS) -D_GNU_SOURCE -I include
 SHARED_FLAGS := -O3 -flto -fPIC -fvisibility=hidden -fno-plt \
-    $(call safe_flag,-fstack-clash-protection) -fstack-protector-strong -pipe -Wall -Wextra \
```

Comment:
Someone will need to look into why safe_flag doesn't work as the Makefile definition is supposed to check if the compiler supports -fstack-clash-protection. arm64 does not support -fstack-clash-protection yet. https://github.com/GrapheneOS/hardened_malloc/issues/149#issuecomment-1010526647

Also, the other arguments _should_ work. I know for a fact -pipe does.